python/lsst/daf/base/propertyContainer/propertyContainerContinued.py

```diff
@@ -170,8 +205,7 @@ def setstate(self, state):
 class PropertySet:
     # Mapping of type to method names
     _typeMenu = {bool: "Bool",
-                 long: "LongLong",
-                 int: "Int",  # overwrites long on Python 3.x
+                 int: "Int",
```

Comment:
Since all Python 3 `int`s are potentially very large, should this mapping be
```
int: "LongLong",
```
?

Reply:
That is an excellent question. I just copied the old code, which would over-wrote the `long: "LongLong"` entry for `int` in Python 3. I'm wondering if @ktlim has an opinion about this. I'm certainly happy to change it.

Reply:
This is not actually used at all now that there's `_guessIntegerType()`.

Reply:
So should I delete the entry or add a comment explaining that the exact integer type doesn't matter as long as it's some kind of integer, or...?

Reply:
I'm pretty sure it isn't used at all, so it's best to delete it to make sure it isn't used accidentally.

Reply:
Thank you. I deleted it with a code comment. The unit tests pass.